clang/lib/Sema/SemaDecl.cpp

```diff
@@ -18377,6 +18377,12 @@ Decl *Sema::getObjCDeclContext() const {
   return (dyn_cast_or_null<ObjCContainerDecl>(CurContext));
 }
 
+Sema::DeviceDiagnosticReason Sema::getEmissionReason(FunctionDecl *FD) {
```

Comment:
This interface really feels like it should accept a `const FunctionDecl *`.

Reply:
Done